config/jobs/kubernetes-sigs/mcs-api/mcs-api-presubmits-master.yaml

```diff
@@ -0,0 +1,123 @@
+presubmits:
+  kubernetes-sigs/mcs-api:
```

Comment:
/assign @skitt 

I'd appreciate any more thoughts on what or what not to include in this first batch of tests or in case you spot any mistakes beyond the one I just fixed :)

Reply:
@skitt Could you PTAL? Thanks!

Reply:
Sorry, this didn’t appear in my review queue! (IME it’s best to /cc people for reviews rather than /assign.)